learning/dashboard/group9/readme.md

```diff
@@ -0,0 +1,105 @@
+# How to install and setup NodeJS and npm?
+- **step 1:**
+Open the Terminal on your Ubuntu system. You can do this by pressing the **"Ctrl+Alt+T"** keys simultaneously.
```

Comment:
nit: Use code block to represent commands or keys 
for ex: `Ctrl+Alt+T`

Reply:
ok done